.github/workflows/check-admin-ui-api.yml

```diff
@@ -0,0 +1,50 @@
+name: Admin UI API Check
+on: [pull_request]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+
+      - name: Check for changes to the APIs the admin UI integration uses
+        id: graphql_inspector
+        run: |
+          npx @graphql-inspector/cli@3.4.0 diff \
```

Comment:
Apparently there is an extra version for CI environment: https://www.the-guild.dev/graphql/inspector/docs/products/ci

I guess we should use that instead of this CLI?